pages/en/lb4/Creating-components.md

```diff
@@ -228,6 +230,33 @@ export class AuthenticationProvider {
 }
 ```
 
+## Extends Application
+
+When bind a component to an app, you may want to mount the component's propertis 
+and methods to application level context. 
+A stragety to do that is using mixin, which extends a class with new properties and methods.
+
+An example scenario would be an app has multiple components with repositories bound to each of them, 
+you can use a `RepositoryMixin` to mount all of the repositories to the app.
+
+If you are not familiar with concept `mixin`, check [Mixin](Mixin.htm) to know more, 
```

Comment:
suggested change.. simply say sth like:
For more details, see [Mixin](Mixin.htm).

Reply:
I would like to have the current version since that document is not a must read for people who already knows JS `mixin`. While a simplified version implies `Mixin.md`contains something additional/different to the JS concept, but it doesn't :)